php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -909,6 +909,7 @@ class NDB_BVL_Instrument extends NDB_Page
                  WHERE testID =:tid
                     AND pass =:cert_id
                     AND centerID =:cid
+                    AND examiners.active='Y'
```

Comment:
What happens here to existing data if there's data already saved and the examiner active flag changes?